src/components/sky-nav/sky-nav.twig

```diff
@@ -0,0 +1,40 @@
+{# https://webaim.org/techniques/skipnav/ #}
+<a href="#main">
+  Skip to main content
+</a>
+{#
+  This may require `role="banner"` if nested in something other than `body`.
+  https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role
+#}
+<header>
+  <a href="/"
+    {% if is_home %}
+      aria-current="page"
+    {% else %}
+      title="Return to home"
```

Comment:
I'm curious if this is needed in addition to the visually hidden text? Would this be redundant for a screen reader?

Maybe instead of adding a title we could conditionally change the visually hidden text?